fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -47,6 +47,7 @@ struct GrvProxyStats {
 
 	LatencyBands grvLatencyBands;
 	LatencySample grvLatencySample;
+    LatencySample grvBatchLatencySample;
```

Comment:
Use tab instead. You can setup automatic `clang-format` as a pre-commit hook to format code.